tools/logging.py

```diff
@@ -8,17 +8,31 @@
 #
 # license: GPLv2
 #
+
+# Standard library imports
 import datetime
-import json
 import os
 import sys
 
+# Third party imports (anything installed into the local Python environment)
+
+# Local application imports (anything from EESSI/eessi-bot-software-layer)
```

Comment:
Should we include these lines if there are no such imports?
Maybe add a `# (none yet)` below each?

Reply:
I thought it would be good to keep them, so in the future if we add imports we put them in one of the categories. Will add the `# (none yet)`.